bin/deploy-prod

```diff
@@ -13,4 +13,4 @@ docker push public.ecr.aws/t1q6b4h2/universal-application-tool:latest
 TIMESTAMP=$(date +%s)
 
 aws s3 sync ./infra s3://seattle-civiform-cftmpl/${TIMESTAMP}
-aws cloudformation update-stack --region ${REGION} --stack-name civiform --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}]"
+aws cloudformation update-stack --region ${REGION} --stack-name civiform --template-url  https://seattle-civiform-cftmpl.s3-${REGION}.amazonaws.com/$TIMESTAMP/stack.yaml --parameters "[{\"ParameterKey\": \"Timestamp\", \"ParameterValue\": \"$TIMESTAMP\"}, {\"ParameterKey\": \"Environment\", \"ParameterValue\": \"prod\"}]"
```

Comment:
Thoughts (non-blocking): Similarly we might be able to use the same script.

Reply:
They're going to need to build the container a little differently once I get around to that, so leaving them separate.